README.md

```diff
@@ -20,9 +20,9 @@ You will need to install it thanks to the `pip` binary.
 
 # Run the tests
 
-## Using a network configuration file
+The network configuration must be defined in a yaml file.
```

Comment:
- What about "is" instead of "must be"?
- What about "described" instead of "defined"?

Reply:
Thanks.